app/controllers/ajax_controller.rb

```diff
@@ -56,6 +56,45 @@ def test
     render(:text => 'test')
   end
 
+
+  #Used by infite scroll
+  def index_rss_log
+    User.current = get_session_user
+    if !params[:type].blank?
+      types = params[:type].split & (["all"] + RssLog.all_types)
+      query = find_or_create_query(:RssLog, type: types.join(" "))
+    else
+      query = find_query(:RssLog)
+      query ||= create_query(:RssLog, :all,
+                             type: @user ? @user.default_rss_type : "all")
+    end
+
+    show_selected_rss_logs(query, id: params[:id].to_s, always_index: true)
+  end
+
+  #used by infinite scroll
+  def show_selected_rss_logs(query, args = {})
+    #store_query_in_session(query)
+    set_query_params(query)
+    args = {
```

Comment:
I suggest running Rubocop on this file to see how Rubocop would like these lines indented.